torch/ao/quantization/fx/backend_config/native.py

```diff
@@ -192,6 +220,14 @@ def _get_linear_configs():
 
     # (3) Linear + batchnorm
     # ------------------------
+    # 3.1 linear bn fusion
+    linear_configs.append({
+        "pattern": (nn.BatchNorm1d, nn.Linear),
+        "dtype_configs": dtype_configs ,
+        "fuser_method": reverse2(fuse_linear_bn)
+    })
+
+    # 3.2 linear bn quantization
```

Comment:
this config is used in fusion, after this is fused, the pattern won't appear in the following qunatization flow, and theoretically we could just remove this config

Reply:
depends on what we mean by quantization I guess, if quantization = fusion + insert observers + convert then yes, both are used in quantization.
if quantization = insert observers + convert then fusion and quantization are separate steps.

Maybe we could have a different name for `insert observers + convert` to avoid overloading "quantization"

Reply:
fusion/fused sounds good I feel, I will make this change

Reply:
fusion configs will be needed, I don't think we can remove them, they are used in the fuse.py function